### PR TITLE
remove dependencies on rosdoc-lite to build

### PR DESCRIPTION
**Pull request by auman66**
### Description

Build directly with the sphinx script. Removes the dependency on rosdoc-lite, which isn't being maintained.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Review by MarqRazz — COMMENTED** [review]
I'm bummed I spent a bunch of time getting rosdoc_lite to work but this is much cleaner! We also need to remove `rosdoc_lite` and `genmsg` from [moveit2_tutorials.repos ](https://github.com/ros-planning/moveit2_tutorials/blob/b24d50a0dd0cb7ed2d5c529e6e61ef29e09d8311/moveit2_tutorials.repos#L10) file

File: moveit2_tutorials.repos [hunk 1]
```diff
@@ -1,8 +1,4 @@
 repositories:
-  moveit2_tutorials:
-    type: git
-    url: https://github.com/ros-planning/moveit2_tutorials.git
-    version: main
```
Comment (MarqRazz) [inline moveit2_tutorials.repos:5]: Why did you remove this repo? This will break the instructions in the README. We only need to remove rosdoc_lite and genmsg from this file.

Comment (auman66) [reply moveit2_tutorials.repos:5]: Your right - my bad
